cmd/deploy/deploy.go

```diff
@@ -278,7 +287,7 @@ func Deploy(ctx context.Context, at AnalyticsTrackerInterface, insightsTracker b
 	cmd.Flags().StringVarP(&options.Namespace, "namespace", "n", "", "overwrites the namespace where the development environment is deployed")
 	cmd.Flags().StringVarP(&options.K8sContext, "context", "c", "", "context where the development environment is deployed")
 	cmd.Flags().StringArrayVarP(&options.Variables, "var", "v", []string{}, "set a variable (can be set more than once)")
-	cmd.Flags().BoolVarP(&options.Build, "build", "", false, "force build of images when deploying the development environment")
+	cmd.Flags().BoolVarP(&options.Build, "build", "", true, "enable/disable building images. Default is true")
```

Comment:
Yep, I also see this weird to set them as `--build=false` in order to make it work.
cc: @pchico83

Reply:
How is thee behavior of `--build` and `--no-build` in other commands? It's also important to keep a consistent syntax across commands

Reply:
In build we have `--no-cache`. We also had `--no-bash` so it was something that we were doing. This `--build` is the only one with this change that is a `boolean` with a default value set to true

Reply:
Let's do `--no-build` then

Reply:
I'd go with the same approach. It's a Boolean variable and it's easier to write --no-wait than setting it to false.

Reply:
This one is tricky, because in the manifest is called `wait`:
https://www.okteto.com/docs/reference/okteto-manifest/#dependencies-string-optional
I would keep `wait` for that reason